Events/Event.swift

```diff
@@ -51,4 +51,8 @@ public class Event<SENDER: AnyObject, INPUT> {
         }
     }
     
+    public func clearEvents() {
```

Comment:
`Event.clearEvents`? It should be `clearListeners`. You even run `listeners.removeAll()` and not `events.removeAll()` ;)